en/flight_modes/follow_me.md

```diff
@@ -1,23 +1,29 @@
 ---
 author: Jimmy Johnson
+author: Junwoo Hwang

```

Comment:
We don't use these keys and the duplicate breaks the build. Propose all this front matter is removed and this is added to acknowledgements heading at the bottom - something like "Thanks to Jimmy Johnson and @junwoo091400". Even that is not strictly necessary.